jigsaw-game/client/tiles.py

```diff
@@ -0,0 +1,67 @@
+from pygame import image, sprite
+
+
+class JigSawTiles(sprite.Group):
+    """Inheritance for the addition of Group methods concerning active tiles"""
+
+    def get_active(self):
```

Comment:
```suggestion
    def get_active(self) -> Tile | False:
```
I don't think this should return `False`. `None` would be better.